test/rspack-build-tests-manifest.json

```diff
@@ -8395,13 +8395,19 @@
   },
   "test/integration/amphtml/test/index.test.js": {
     "passed": [
+      "AMP Usage production mode should not contain missing files warning",
+      "AMP Usage production mode should not have deprecation warning"
+    ],
+    "failed": [
       "AMP Usage production mode With AMP context should render nested AMP page with AMP hook",
       "AMP Usage production mode With AMP context should render nested normal page with AMP hook",
```

Comment:
Probably that Webassembly loading flakeyness we always get

Reply:
Rspack doesn't currently have any retries for CI capacity reasons (lots of test timeouts were causing problems), but we can probably add them back now that most tests are passing.